content/en/monitors/guide/monitor-for-value-within-a-range.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Monitor if a value is inside or outside a range
+kind: guide
+---
+
+## Overview
+
+If monitors supports to alert when a given value is above or below a given threshold, it is also possible to be notified if a given value is inside or outside of a given range.
+
+### Metric Example
+
+Let's imagine that a metric `a` reports discrete values from `0` to `10` representing a status and you want to be notified if the metric is not between `4` and `8`.
+Mathematically, this is the same as saying that the difference between the metric and the center of the range (6) should never be more than 2. 
+
+`8 > a > 4` <=> `abs(6-a) < 2` <=> `abs(6-a) - 2 < 0`
```

Comment:
I recommend putting this in a code block

Reply:
done :)